documentation/image_duplicates.md

```diff
@@ -0,0 +1,167 @@
+# Tutorial: Finding Image Duplicates using the ImageDuplicates Feature Extractor
+In this tutorial, we will explore how to utilize the ImageDuplicates class to identify duplicate images within a specified directory containing images. The ImageDuplicates class is a feature extractor that employs Difference Hashing to detect duplicate images based on their hash codes. We will guide you through the process of initializing and using this class to find duplicates within the COCO2017 detection dataset and examine the results.
+
+
+## Step 1: Adding the ImageDuplicates to the Image Features Report Section Features List
+Just like any other feature extractor, you need to include it in your feature list within the appropriate report section.
+Since finding duplicates is not task-specific, it belongs in the "Image Features" report section. In your detection.yaml configuration file, add it as follows:
```

Comment:
I would state that this feature is the only that is not in the default features because it requires explicit `train_image_dir`, while other features dont require to change the `.yaml`

Reply:
Added in latest commit.